views/Admin/users.html.twig

```diff
@@ -0,0 +1,29 @@
+{% extends 'Admin/layout.html.twig' %}
+{% block content %}
+    <header class="jumbotron subhead" id="overview">
+        <h1><i class="glyph2x-group"></i>Users</h1>
+        <p class="lead pull-left">Manage Baïkal user accounts, and associated resources.</p>
+        <p class="lead pull-right"><a class="btn btn btn-inverse" href="#"><i class="icon-white icon-user"></i> + Add user</a></p>
+    </header>
+
+    <table class="table table-bordered table-striped users">
+        {% for user in users %}
+            <tr>
+                <td class="col-username">
+                    <i class="icon-user"></i> <strong>{{ user.username|escape }}</strong><br />
+                    {{ user.displayname|escape }} <a href="{{ user.mailtouri|escape }}">&lt;{{ user.email|escape }}&gt;</a>
+                </td>
+                <td class="col-actions no-border-left">
+                    <p class="pull-right">
+                        <nobr><a class="btn" href="{{ user.linkcalendars }}"><i class="{{ calendaricon }}"></i> Calendars</a></nobr>
```

Comment:
For the most, yes. I did copy it and ajust it a bit to fit into the new format (template extends, removing or renaming some twig variables, etcetera).

One of the biggest choices now is implementing custom forms vs using form builders (csfr included or customly implemented) or whether to use an ORM (if so, Doctrine or something else?) and at least abstracting implementation away now using the Repository pattern.